README.md

```diff
@@ -69,10 +69,30 @@ AWS command line utility and configure your AWS access credentials.
    that `AWS_DEFAULT_REGION` points to your prefered region.
    These buckets will be created with Terraform, and should not exist before deploying for the first time.
 
+1. Create an AWS ElasticSearch Service instance with the name specified by `$DSS_ES_DOMAIN` (e.g. via `aws es create-elasticsearch-domain --domain-name $DSS_ES_DOMAIN`).
+
 #### Configure GCP
 
 1.  Follow the instructions in https://cloud.google.com/sdk/downloads to get the `gcloud` command line utility.
 
+1.  Generate service account keys for setup:
+
+    1) Go to https://console.developers.google.com/apis/credentials (you may have to select Organization and Project
+    again)
+    
+	1) Click *Create Credentials* and select *Service account key*
+    
+	1) For *Key type* choose JSON. 
+    
+
+	1) Place the downloaded JSON file into the project root as `gcp-credentials.json` (or whatever `GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME` is set to in your `environment.local`).
```

Comment:
Try to use a more formal language for a dev doc (or whatever ->  to what the); with that being said i think telling the user that placing it in the root for the project is suffice, in the next code command below this, the user is told to `cat $DSS_HOME/gcp-credendials.json` but if they move the file around following the statement above, that wont work correctly.